src/utils/parse-comment/index.js

```diff
@@ -57,7 +57,7 @@ const plugin = {
 nlp.plugin(plugin)
 
 function parseAddComment(doc, action) {
-    const who = doc
+    let who = doc
```

Comment:
[minor, not blocking] usually try to keep variables assignment immutable with `const`
see https://medium.com/dailyjs/use-const-and-make-your-javascript-code-better-aac4f3786ca1